iot_config/src/region.rs

```diff
@@ -0,0 +1,128 @@
+use helium_proto::Region as ProtoRegion;
+use serde::{de, Deserialize, Deserializer, Serialize};
+use std::{fmt, str::FromStr};
+
+#[derive(Clone, Debug, Eq, Hash, Ord, PartialEq, PartialOrd)]
+pub enum Region {
```

Comment:
It kills me that we define these regions defined in so many places. I'm not sure what the best solution is.

Reply:
can we use the photo region enum as the single source of truth?

Reply:
+1 ^

Reply:
full disclosure, i cargo culted (ha!) this module as well before i was really using it but we can't use the proto enum out of the box b/c we need the proto region to implement `Deserialize` in order to implement serde for the `GwmpMap` type in the route module but if i'm able to update this module to just wrap the proto enum in a new type or barring that, implement Serialize and Deserialize for the region _in_ the proto repo then we can at least simplify this

Reply:
We should be able to derive serialize for the region struct.

Reply:
there are pattern matches and directives for Prost to declare attributes for structs

Reply:
Lemme give that a shot then. That would certainly be the cleanest solution

Reply:
unless there's a more succinct way of defining the derive for deserialize on the region proto, https://github.com/helium/proto/pull/259 does appear to do the trick; i've eliminated the local region module entirely as a result in this latest commit